docs/docs/general/03-Guides/reporter.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 8
+---
+
+# AskUI Reporter
+We provide a package `askui-reporters` with reporters ready-to-use to create reports for your AskUI runs.
+
+On this page you will learn how to use the Allure-reporter from this package and how you can implement your own reporter for your reporting tool.
```

Comment:
As a reader I think I would be missing that info:

- the more general context that `@askui/askui-reporters` is made up of third-party integrations of reporters ready to be integrated with askui lib, reporting utils
- Explanation of what is `Allure` 
- AskUI recommends using Allure + why we recommend it

I think it would be enough to move the link the repo to the first paragraph.